starter-code/server.js

```diff
@@ -34,15 +37,15 @@ app.use(express.static('./public'));
 // REVIEW: Routes for requesting HTML resources
 app.get('/new', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here: This corresponds to #1, Create view. This is the '.toHtml' function. This is the 'C' part of CRUD, or Create.
   response.sendFile('new.html', {root: './public'});
 });
 
 
 // REVIEW: Routes for making API calls to use CRUD Operations on our database
 app.get('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here: This corresponds to #2, request. This is the '.loadAll' function. This is the 'R' part of CRUD, or Read.
```

Comment:
You're right, this is the R in CRUD, but look back at the code, is loadAll method being called?